test/tutorials/combinatorial/test_combinatorial_tutorials.py

```diff
@@ -0,0 +1,34 @@
+"""Unit tests for the tutorials."""
+
+import glob
+import subprocess
+import tempfile
+
+import pytest
+
+
+def _exec_tutorial(path):
```

Comment:
We should not copy-paste the function exec_tutorial. 

Instead, it is OK not to have one test folder per tutorial's folder, because the test file does not have unit-tests anyway: it only runs the tutorials.

You can bring back test_tutorials.py, and do:
```
paths = sorted(glob.glob("tutorials/combinatorial/*.ipynb"))
paths.extend(sorted(glob.glob("tutorials/combinatorial/*.ipynb")))
....
etc
````